etc/docker/Dockerfile.nvcr-triton-containers

```diff
@@ -0,0 +1,78 @@
+ARG TRITON_VERSION=24.06
+ARG JETPACK
```

Comment:
This way, you can:
```
docker build ... --build-arg JETPACK=1 ...
```
To get a Jetpack targeted build, and
```
docker build ...
```
To get a non-JP one.